### PR TITLE
Empty storage option for isoltest.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: test/libsolidity/SemanticTest.cpp [hunk 1]
```diff
@@ -176,7 +176,17 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 				constructed = true;
 			}
 
-			if (test.call().kind == FunctionCall::Kind::Constructor)
+			if (test.call().kind == FunctionCall::Kind::Storage)
+			{
+				test.setFailure(false);
+				bool nonEmpty = !storageEmpty(m_contractAddress);
+				bytes result(1, nonEmpty);
```
Comment (mijovic) [inline test/libsolidity/SemanticTest.cpp]: Only used here, so maybe you can remove the variable: 
```suggestion
				bytes result(1, !storageEmpty(m_contractAddress));
```

File: test/libsolidity/util/TestFileParser.cpp [hunk 2]
```diff
@@ -85,6 +85,21 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 						call.kind = FunctionCall::Kind::Library;
 						call.expectations.failure = false;
 					}
+					else if (accept(Token::Storage, true))
+					{
+						expect(Token::Colon);
+						call.expectations.failure = false;
+						call.expectations.result.push_back(Parameter());
+						// empty / non-empty is encoded as false / true
```
Comment (mijovic) [inline test/libsolidity/util/TestFileParser.cpp:93]: Would it be more intuitive if `empty` is encoded to `true`?

**Comment by mijovic** [comment]
Can you add test where storage goes from `empty` to `non-empty` and end the end back to `empty`?

File: test/libsolidity/util/TestFileParser.cpp (hunk 2 above)
Comment (chriseth) [reply test/libsolidity/util/TestFileParser.cpp:93]: I thought it makes sense that `0` is encoded as empty... ("set to zero")

**Review by mijovic — APPROVED** [review]

**Pull request merged**